java/opendcs/src/main/java/decodes/rledit/EUTableModel.java

```diff
@@ -52,11 +52,14 @@ public class EUTableModel extends AbstractTableModel
 	private ArrayList<EngineeringUnit> eus;
 	private int lastSortColumn;
 
+	Database db;
+
 	/**
 	 * No args constructor for JBuilder.
 	 */
 	public EUTableModel()
 	{
+		this.db = Database.getDb();
```

Comment:
In the constructor would be better, but this is still a good improvement.

Also EngineeringUnitsList is a good candidate for migration to the new DAOs. While used just about everywhere it's contents are fairly constrained.

Reply:
you would have to create the DAO and change all the usages. i wasn't suggesting to do it for this PR.